server/model/models.go

```diff
@@ -51,6 +56,34 @@ type File struct {
 // Role represents the position of a app User
 type Role string
 
+// Value returns the string value of the Role
+func (r Role) Value() (driver.Value, error) {
+	// TODO: role validation (@dpordomingo)
+	return string(r), nil
+}
+
+// Scan sets the Role with the passed string
+func (r *Role) Scan(value interface{}) error {
+	// TODO: role validation (@dpordomingo)
+	if value == nil {
+		*r = Role(Worker)
+		return nil
+	}
+
+	var bv driver.Value
+	var err error
+	if bv, err = driver.String.ConvertValue(value); err == nil {
```

Comment:
why so complicated? https://play.golang.org/p/3ynyW8ZGKJ5 covers everything we need, no?